script.js

```diff
@@ -9,4 +9,129 @@ hamburger.addEventListener('click', () => {
 document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
   hamburger.classList.remove('active');
   navMenu.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+const projects = [
```

Comment:
When the user clicks (or taps) the button to check project details, the popup with details about the project appears ✔

Reply:
When the user clicks (or taps) the close (X) button, the popup disappears 🔥